content/docs/use-cases/versioning-data-and-model-files/tutorial.md

```diff
@@ -33,8 +33,8 @@ model file.
 
 ## Preparation
 
-> We have tested our tutorials and examples with Python 3. We don't recommend
-> using earlier versions.
+> We have tested our tutorials and examples with Python 3.7. We don't recommend
+> using earlier or later versions.
```

Comment:
Maybe just
```suggestion
> We have last tested this tutorial with Python 3.7.
```
But I need to confirm this info. It's quite possible, as this one tutorial hasn't been tried in a while...